pkg/utils/perm_windows.go

```diff
@@ -6,6 +6,21 @@
 
 package utils
 
+// FileOwner is the ownership a file should have.
+type FileOwner struct {
+	UID string
+	GID string
+}
+
+// CurrentFileOwner returns the executing UID and GID of the current process.
+func CurrentFileOwner() FileOwner {
+	// TODO(blakerouse): Make this return the current user and group on Windows.
+	return FileOwner{
+		UID: "S-1-5-32-544", // Administrator
+		GID: "S-1-5-18",     // SYSTEM
```

Comment:
may be worth extracting to consts and using these consts in `internal/pkg/agent/install/perms_windows.go` as well. helps tracking usage

Reply:
Unified this in the PR to use a constant here and anywhere else that used those same strings. It was in a few places before this change.